src/client.ts

```diff
@@ -420,13 +426,15 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
    * Disconnect the streaming connection
    */
   public disconnect (): Promise<any> {
+    this._http.stopAllRetries();
     return this._streamingConnection.disconnect();
   }
 
   /**
    * Reconnect the streaming connection
    */
   public reconnect (): Promise<any> {
+    this._http.stopAllRetries();
     return this._streamingConnection.reconnect();
   }
 
```

Comment:
We can actually junk this whole refreshIceServers function and everything that relates to it. it moved up to streaming client.

Reply:
If we remove the whole thing, I would like to do that in v6 to avoid any breaking changes. I know this isn't a publicly advertised method, but I think it would still be safer to do it that way.